keys.go

```diff
@@ -233,6 +233,46 @@ func (c *Client) DeleteKey(ctx context.Context, id string, prefer PreferReturn,
 	return nil, nil
 }
 
+// RestoreKey restores a deleted imported root key by specifying the ID of the key
+func (c *Client) RestoreKey(ctx context.Context, id, payload, encryptedNonce, iv string, prefer PreferReturn) (*Key, error) {
```

Comment:
You aren't using the prefer return anywhere... please remove it.

Did you copy paste this from somewhere?

Reply:
I do want to pass in the return representation. I see that I need to add to the request header.